module/signer.go

```diff
@@ -4,13 +4,15 @@ import (
 	"github.com/onflow/flow-go/crypto"
 )
 
+// TODO : to delete in V2
```

Comment:
I think we still need the `Signer`, the consensus follower is able to verify and sign a block in a contextless manner
```suggestion
```

Reply:
I mean we don't need that exact structure there